demos/checkbox.html

```diff
@@ -201,6 +202,10 @@
         document.getElementById('make-ind').addEventListener('click', function() {
           checkboxInstance.indeterminate = true;
         });
+
+        document.getElementById('toggle-disabled').addEventListener('click', function() {
```

Comment:
We will make those decisions on a component by component basis, but it has come up more than just here. Toolbars are another example where we are considering something like this.

If you wouldn't mind adding these to the checkbox I'll give it another review! Thanks